documentation/builders/webapp/playlists-livestreams-podcasts.md

```diff
@@ -0,0 +1,141 @@
+4. You should now be able to browse and play the content of the playlist.
+
+#### Example folder structure
+
+```
+└── audiofolders
+    ├── wake-up-songs
+    │   └── playlist.m3u
+    └── lullabies-sleep-well
+        └── playlist.m3u
+```
+
+### Assigning a .m3u playlist to a card
+
+In the Phoniebox Web App, .m3u playlists do not show up as individual files. In order to assign a playlist to a card, do the following:
+
+1. [Follow the steps above](#using-m3u-playlist-in-phoniebox) to add a playlist to your Phoniebox (make sure you have created individual folders).
+2. In the Web App, open Cards tab and click the "+" button to add a new card.
+3. As a Jukebox action, select "Play music", then select "Select music".
+4. Next (in your your Library view), select the Folder view in the top right corner.
+5. Browse to the folder you created (representing your playlist) and click on it.
+
+You are essentially assigning a folder (just like any other conventional folder) to your card representing the content of your playlist.
+
+## Livestreams
+
+In order to play radio livestreams on your Phoniebox, you use playlists to register your livestream and make it accessible.
+
+### Using livestream.txt playlist in Phoniebox
+
+1. [Follow the steps above](#using-m3u-playlist-in-phoniebox) to add a playlist to your Phoniebox (make sure you have created individual folders).
+2. When creating the playlist file, make sure it call or at least ends with `livestream.txt` instead of `*.m3u` (Examples: `awesome-livestream.txt`, `livestream.txt`).
+3. Add links to your livestreams just like you would with songs in .m3u playlists
+
+You can now assign livestreams to cards [following the example](#assiging-a-m3u-playlist-to-a-card) of playlists.
+
+#### Example folder structure and playlist names
+
+```
+└── audiofolders
+    ├── wdr-kids
+    │   └── wdr-kids-livestream.txt
+    ├── energy
+    │   └── cool-livestream.txt
+    └── classic
+        └── livestream.txt
+```
+
+#### Example of livestream.txt
+
+```txt
+https://wdr-diemaus-live.icecastssl.wdr.de/wdr/diemaus/live/mp3/128/stream.mp3
+http://channels.webradio.antenne.de/hits-fuer-kids
+```
+
+## Podcasts
+
+Just like you add livestreams to the Phoniebox, you can also add individual Podcasts or entire Podcast feeds to the Phoniebox.
+
+You have 3 options to play Podcasts
+
+1. Create a playlist and reference individual direct links to Podcast episodes (just like livestreams)
+2. Provide a Podcast RSS feed
+3. Download the MP3 and add them like normal songs to your Phoniebox. This makes them available offline.
+
+We will explain options 1 and 2 more closely.
+
+### Using podcast.txt playlist in Phoniebox
+
+1. [Follow the steps above](#using-m3u-playlist-in-phoniebox) to add a playlist to your Phoniebox (make sure you have created individual folders).
+2. When creating the playlist file, make sure it ends with `podcasts.txt` instead of `*.m3u`. (Examples: `awesome-podcast.txt`, `podcast.txt`).
```

Comment:
Harmonized with description above
```suggestion
2. When creating the playlist file, make sure it is called or at least ends with `podcasts.txt` instead of `*.m3u`. (Examples: `awesome-podcast.txt`, `podcast.txt`).
```